sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -126,9 +130,15 @@ def _create_hierarchical_data():
 @pytest.mark.parametrize("scoring", TEST_METRICS)
 @pytest.mark.parametrize("cv", CVs)
 @pytest.mark.parametrize("error_score", ERROR_SCORES)
-def test_gscv(forecaster, param_grid, cv, scoring, error_score):
+@pytest.mark.parametrize("multivariate", [True, False])
+def test_gscv(forecaster, param_grid, cv, scoring, error_score, multivariate):
     """Test ForecastingGridSearchCV."""
     y, X = load_longley()
```

Comment:
This line is redundant given the condition next line

Reply:
Will you delete it? I still can see it here

Reply:
yes - this line is truly invisible, hiding among the other longleys.